apis/supernova/src/qae.js

```diff
@@ -66,8 +66,11 @@ const resolveValue = (data, reference, defaultValue) => {
 function target(def) {
   const propertyPath = def.path || '/qHyperCubeDef';
   const layoutPath = propertyPath.slice(0, -3);
-  if (/\/qHyperCube$/.test(layoutPath) === false) {
-    throw new Error(`Incorrect definition for qHyperCubeDef at ${propertyPath}`);
+  if (/\/(qHyperCube|qListObject)$/.test(layoutPath) === false) {
+    const d = layoutPath.includes('/qHyperCube') ? 'qHyperCubeDef' : 'qListObjectDef';
+    throw new Error(
+      `Incorrect definition for ${d} at ${propertyPath}. Valid paths include /qHyperCubeDef or /qListObjectDef, e.g. data/qHyperCubeDef`
+    );
```

Comment:
consider using slightly more generic error message when that path don't include HyperCube or ListObject.
(maybe just Incorrect definition at ${propertyPath}. Valid paths include /qHyperCubeDef or /qListObjectDef, e.g. data/qHyperCubeDef)
(if we want a default I think qHyperCubeDef would be a better alternative then qListObjectDef)